internal/workload/podspec_updates_test.go

```diff
@@ -916,6 +915,61 @@ func TestPodTemplateAnnotations(t *testing.T) {
 
 }
 
+func TestTelemetryAddsTelemetryContainerPort(t *testing.T) {
+
+	var (
```

Comment:
Go nit: are we planning on declaring more variables? Otherwise, why not `var u = ...`?

Reply:
Fixed